src/main/java/rs/hybridit/challenges/Challenges.java

```diff
@@ -19,27 +19,27 @@ public boolean checkIfStringIsAPalindrome(String text) {
     }
 
     @Override
-    public Object findDuplicateCharacters(String text) {
-        HashMap<Character, Integer> mapa = new HashMap<Character, Integer>();
+    public String findDuplicateCharacters(String text) {
+        HashMap<Character, Integer> map = new HashMap<Character, Integer>();
         for (int i = 0;i < text.length(); i++){
```

Comment:
Use intellij format code, Code -> Reformat code, and remove unnecessary empty lines (there are 4-5 lines in some places). This will make code more readable